app/code/Magento/Swatches/Test/Mftf/Test/AdminDisablingSwatchTooltipsTest.xml

```diff
@@ -18,145 +18,81 @@
             <group value="Swatches"/>
         </annotations>
         <before>
-            <!-- Create category -->
             <createData entity="ApiCategory" stepKey="createCategory"/>
 
-            <!-- Log in -->
+            <createData entity="ApiConfigurableProductWithDescription" stepKey="createProduct">
+                <requiredEntity createDataKey="createCategory"/>
+            </createData>
+
+            <createData entity="VisualSwatchProductAttributeForm" stepKey="createVisualSwatchAttribute"/>
+            <createData entity="SwatchProductAttributeOption1" stepKey="swatchAttributeFirstOption">
+                <requiredEntity createDataKey="createVisualSwatchAttribute"/>
+            </createData>
+
             <actionGroup ref="AdminLoginActionGroup" stepKey="loginAsAdmin"/>
         </before>
         <after>
-            <!-- Clean up our modifications to the existing color attribute -->
```

Comment:
Here as well, if we remove the existing step keys, we introduce backward-incompatible changes.